libqtile/widget/windowname.py

```diff
@@ -35,6 +35,8 @@ class WindowName(base._TextBox):
         ('show_state', True, 'show window status before window name'),
         ('for_current_screen', False, 'instead of this bars screen use currently active screen'),
         ('empty_group_string', ' ', 'string to display when no windows are focused on current group'),
+        ('max_chars', 0, 'max chars before truncating with ellipsis'),
+        ('format', '{state}{class}:{name}', 'format of the text'),
```

Comment:
Could we set the default format to the format that it was previously, so that people who are happy with the old form don't have to go in and change their format? (ignore me if this is already the case)